connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -1351,10 +1351,16 @@ private[sql] object Column {
   }
 
   @DeveloperApi
```

Comment:
This is under `private[sql]` so it can't be an API actually @tomvanbussel @hvanhovell

Reply:
Should probably directly remove them.

Reply:
Let's figure this out in a follow-up PR if possible. I'd like to get this merge to unblock some follow-up work.